pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00_test.go

```diff
@@ -393,8 +393,11 @@ func TestAvailableStatus(t *testing.T) {
 			if condition.Status != tc.expectedStatus {
 				t.Error(diff.ObjectGoPrintSideBySide(condition.Status, tc.expectedStatus))
 			}
-			if tc.expectedReason != "" && condition.Reason != tc.expectedReason {
-				t.Error(diff.ObjectGoPrintSideBySide(condition.Reason, tc.expectedReason))
+			expectedReasons := strings.Join(tc.expectedReasons, "\n")
+			if len(expectedReasons) > 0 && condition.Reason != expectedReasons {
+				{
```

Comment:
no need to nest this block any further I think

Reply:
Fixed.